index.css

```diff
@@ -29,53 +30,70 @@ a {
 	padding: 0 1.8vw 0 1.5vw;
 }
 
-/******** grid ***********/
+/********* social media logos ******/
+
+
+.socialWrapper {
+	display:flex;
+	justify-content:flex-end;
+	float:none;
+}
+
+.socialWrapper > a {
+	/* social links */
+	float: right;
+	margin-right: 0.5rem;
+	margin-top: 0.5rem;
+}
+
+.socialWrapper > span {
+	/* email address */
+	font-weight: 400;
+	font-size: 0.7rem;
+	margin-top:0.8rem;
+	margin-right: 0.5rem;
+}
+
+.socialWrapper > a > img {
+	/* social logos */
+	width: 1rem;
+	height: 1rem;
+}
+
+/******** grid's boxes ***********/
 
 .Grid > *,
 .InfoWrapper-textWrapper,
 .BookingContainer-box {
```

Comment:
```css
.Grid > *,
.Box {
```

if you use something like .Box it's clearer what the intent outside of grid. And easier to reuse